server/config/main.ini.dist

```diff
@@ -9,6 +9,9 @@ port = 443
 [mongo]
 dsn = mongodb://user:password@host:27017/feat
 
+[build]
+instancesDirectory = /home/${user}/buildInstances
```

Comment:
Could we use some default that will point to a subdirectory inside the project root directory?